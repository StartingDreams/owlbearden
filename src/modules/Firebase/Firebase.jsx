import React from 'react';
import Init from './containers/Init';
import Auth from './containers/Auth';


export class Firebase extends React.Component {
  componentDidMount = () => {};

  render = () => (
    <div>
      <Init />
      <Auth />
    </div>
  );
}

export default Firebase;
