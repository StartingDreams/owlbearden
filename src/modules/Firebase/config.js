const config = {
  apiKey: 'AIzaSyDg4Z-8ROBMuRtzU5ThnU1GMd2IWgsttEU',
  authDomain: 'owlbearsden.firebaseapp.com',
  databaseURL: 'https://owlbearsden.firebaseio.com',
  projectId: 'owlbearsden',
  storageBucket: 'owlbearsden.appspot.com',
  messagingSenderId: '149275717879',
};

export default config;
