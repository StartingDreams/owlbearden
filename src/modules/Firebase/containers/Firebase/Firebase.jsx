import React from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import firebase from 'firebase';
import { withRouter } from 'react-router-dom';
import config from '../../config';

firebase.initializeApp(config);

export class Firebase extends React.Component {
  componentDidMount = () => {};

  render = () => null;
}

Firebase.propTypes = {
  dispatch: PropTypes.func,
  history: PropTypes.shape({
    push: PropTypes.func.isRequired,
  }).isRequired,
};

export default withRouter(connect()(Firebase));
