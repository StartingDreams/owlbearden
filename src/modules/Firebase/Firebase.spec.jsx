import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Firebase } from './Firebase';

jest.mock('./containers/Init', () => (function Init() {}));
jest.mock('./containers/Auth', () => (function Auth() {}));

describe('<Firebase />', () => {
  it('spell should match the snapshot', () => {
    const wrapper = shallow(<Firebase />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
