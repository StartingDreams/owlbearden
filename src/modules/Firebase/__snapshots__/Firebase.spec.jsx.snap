// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Firebase /> spell should match the snapshot 1`] = `
<div>
  <Init />
  <Auth />
</div>
`;
